internal/checksengine/facts/packageversion.go

```diff
@@ -0,0 +1,37 @@
+package facts
+
+import (
+	"os/exec"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	PackageVersionFactKey = "package_version"
+)
+
+type packageVersionConfigGatherer struct {
+}
+
+func NewPackageVersionConfigGatherer() *packageVersionConfigGatherer {
+	return &packageVersionConfigGatherer{}
+}
+
+func (s *packageVersionConfigGatherer) Gather(packages []string) ([]*Fact, error) {
+	var facts []*Fact
+	log.Infof("Starting Package versions facts gathering process")
+
+	for _, packageName := range packages {
+		version, _ := exec.Command("rpm", "-q", "--qf", "%{VERSION}", packageName).Output()
```

Comment:
I know this is a spike, but for the actual thing, would it make sense to consume directly the rpmdb instead of using the `rpm` command? There is e.g. -> https://github.com/knqyf263/go-rpmdb (which oddly, @dirkmueller already contributed to :smile: )